crypto/rust/src/lib.rs

```diff
@@ -615,6 +615,8 @@ impl_hash_type!(VRFVkey, 32);
 impl_hash_type!(KESVkey, 32);
 // same for this signature (but lots of traits aren't implemented for [u8; 448] so we can't)
 //impl_hash_type!(KESSignature, 448);
+// this might actually be one of the other hashes
```

Comment:
Does anyone know if this is some other existing hash (block hash, etc) or its own thing? We don't use any existing hash in the old CML code (but the Nonce object has been manually edited there).